rotkehlchen/chain/ethereum/modules/eth2/structures.py

```diff
@@ -218,7 +218,7 @@ def process(
             accounting: 'AccountingPot',
             events_iterator: Iterator['AccountingEventMixin'],  # pylint: disable=unused-argument
     ) -> int:
-        if self.pnl_balance == ZERO:
+        if self.pnl_balance.amount == ZERO:
             return 1
 
         amount = self.pnl_balance.amount
```

Comment:
You could have moved the amount variable declared before the if statement but is fine

Reply:
hmm fuck yeah that would have been prettier. Oh well next PR :smile:

Reply:
Actually changed it since more changes had to be added